docs-src/concepts/what-are-actvitiy-events.md

```diff
@@ -0,0 +1,28 @@
+---
+id: what-are-activity-events
+title: What are Activity Events?
+sidebar_label: Activity Events
+description: There are seven Activity Events that are added to History at different points in an Activity Execution.
+tags:
+  - term
+  - explanation
+---
+
+There are seven Activity-related [Events](/workflows#event) that are added to History at different points in an Activity Execution.
+
+There are two important things to note when matching Activity Execution lifecycle to Activity Events:
+
+- In Event names, "ActivityTask" refers to an [Activity Execution](/activities#activity-execution), not an [Activity Task](/tasks#activity-task).
+- While the Activity is running and retrying, [ActivityTaskScheduled](/references/events#activitytaskscheduled) is the only Activity-related event in History: [ActivityTaskStarted](/references/events#activitytaskstarted) is written along with a terminal event like [ActivityTaskCompleted](/references/events#activitytaskcompleted) or [ActivityTaskFailed](/references/events#activitytaskfailed).
+
+The Activity-related Events and points at which they're added to History are:
+
+- Once a [Workflow Task Execution](/concepts/what-is-an-activity-task-execution) reaches a line of code that starts/executes an Activity, the Worker sends the Activity type and arguments to the Cluster, and the Cluster adds an [ActivityTaskScheduled](/references/events#activitytaskcompleted) Event to History.
+- When ActivityTaskScheduled is added to History, the Cluster adds a corresponding Activity Task to the Task Queue.
+- The Worker polling that Task Queue picks up the Activity Task and runs the Activity function or method.
+- If the Activity function returns, then the Cluster adds [ActivityTaskStarted](/references/events#activitytaskstarted) and [ActivityTaskCompleted](/references/events#activitytaskcompleted) to History.
```

Comment:
just being super pedantic, but you might want to make it clear that the worker has to send a message back to the cluster that the activity has completed

Reply:
```suggestion
- If the Activity function returns, then the Worker reports completion to the Cluster, and the Cluster adds [ActivityTaskStarted](/references/events#activitytaskstarted) and [ActivityTaskCompleted](/references/events#activitytaskcompleted) to History.
```